MAUI/ComboBox/Getting-Started.md

```diff
@@ -0,0 +1,241 @@
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+**Step 2:** Populate data in `ComboBox`. 
+
+Now, populate this SocialMediaViewModel data in `ComboBox` control by binding it to the `ItemSource` property. 
+
+{% tabs %}
+{% highlight xaml %}
+
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:editors="clr-namespace:Syncfusion.Maui.Inputs;assembly=Syncfusion.Maui.Inputs"
+             xmlns:local="clr-namespace:ComboBoxSample"             
+             x:Class="ComboBoxSample.MainPage">
+
+       <ContentPage.BindingContext>
+            <local:SocialMediaViewModel />
+       </ContentPage.BindingContext>
+
+       <ContentPage.Content>
+            <!--Setting ItemsSource-->
+            <editors:SfComboBox x:Name="comboBox" 
+                                Width="250"
+                                ItemsSource="{Binding SocialMedias}" />
+        </ContentPage.Content>
+</ContentPage>
+
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+SocialMediaViewModel socialMediaViewModel = new SocialMediaViewModel();
+comboBox.BindingContext = socialMediaViewModel;
+comboBox.ItemsSource = SocialMediaViewModel.SocialMedias;
+
+{% endhighlight %}
+{% endtabs %}
+
+N> Set the SocialMediaViewModel instance as the BindingContext of your control; this is done to bind properties of SocialMediaViewModel to `ComboBox`.
+
+**Step 3:** Setting TextMemberPath and DisplayMemberPath.
+
+The `ComboBox` control is populated with a list of social media. But the SocialMedia model contains two properties, ID and Name, so it is necessary to intimate by which property it should display the value in the selection box portion of the `ComboBox` control when an item is selected.
+
+`TextMemberPath` - This property path is used to get the value for displaying in the selection box portion of the `ComboBox` control when an item is selected. The default value is `String.Empty`.
+
+`DisplayMemberPath` - This property path is used to the name or path of the property displayed for each data item in the drop-down list. The default value is `String.Empty`.
+
+{% tabs %}
+{% highlight xaml %}
+
+<editors:SfComboBox x:Name="comboBox" 
+                    DisplayMemberPath = "Name"
+                    TextMemberPath = "Name"
+                    ItemsSource="{Binding SocialMedias}" />
+
+{% endhighlight %}
+{% highlight C# %}
+
+comboBox.DisplayMemberPath = "Name";
+comboBox.TextMemberPath = "Name";
+
+{% endhighlight %}
+{% endtabs %}
+
+The following gif image illustrates the result of the above code.
+
+![.NET MAUI ComboBox populating using data binding](Images/GettingStarted/ComboBoxGettingStarted.gif)
+
+## Editing
+
+The `ComboBox` control supports editable and non-editable modes to choose items. To enable editing functionality, set the `IsEditable` property as `true`. The default value is `false`.
```

Comment:
The `ComboBox` control supports editable and non-editable modes to choose items. To enable the editing functionality, set the `IsEditable` property as `true`. The default value is `false`.

Reply:
Changes are made.